command/base_predict_test.go

```diff
@@ -538,6 +539,8 @@ func TestPredict_PathsKVv2(t *testing.T) {
 	client, closer := testVaultServerWithKVVersion(t, "2")
 	defer closer()
 
+	time.Sleep(250 * time.Millisecond)
```

Comment:
Was this test flaky or something?

Reply:
Yeah, thought I added a comment, but perhaps not.

The test flakes when I run on power saving mode on my laptop, because it is running so many tests in parallel and the KVv1->KVv2 auto-migration that is done on mounting isn't complete by the time the subsequent write (below) is performed. Cleaner would be to add a retry loop to `testVaultServerWithKVVersion` to wait until the KVv2 migration has been performed, but I'm (a) lazy and (b) there's plenty of other command tests that an extra 250ms among friends won't be missed. :-)

You're welcome to tackle that if you want, I think there's other examples of retry loops (e.g., in PKI tests) that you could use if you want. :D